grails-app/conf/BuildConfig.groovy

```diff
@@ -38,7 +38,7 @@ grails.project.dependency.resolution = {
         runtime 'net.sf.opencsv:opencsv:2.3'
         runtime "org.apache.solr:solr-solrj:5.4.0"
         runtime "org.gbif:dwca-io:1.24"
-        runtime("au.org.ala:ala-name-matching:2.3.1"){
+        runtime("au.org.ala:ala-name-matching:2.3.3-SNAPSHOT"){
```

Comment:
Should be. Once the new BIE is up and running on AWS the version numbers on a bunch of things BIE-y should be bumped before deployment on the proper servers.